### ISSUE TITLE
Generated code doesn't consider default types in the schema

### ISSUE THREAD

**Issue by DimaMukhin** [dotansimha/graphql-code-generator#1007]
**Describe the bug**
Default values in the schema are ignored when generating types. A generated type will be nullable, even if the type has a default value in the schema. See example below for more details.

schema.graphql
```
input OffsetPaginationInput {
  offset: Int = 0
  limit: Int = 10
}
```

generated-types.ts
```
export interface OffsetPaginationInput {
  offset?: number | null;
  limit?: number | null;
}
```

expected generated types to be:
```
export interface OffsetPaginationInput {
  offset?: number;
  limit?: number;
}
```

**To Reproduce**
1. create a schema file with the input type from the example above
2. generate types with `gql-gen`

**Expected behavior**
If types have a default value, they cannot be null. Therefore, I expect the generated type to be non-nullable. Even if the type is optional.

**Schema/Documents**
Please see examples above

**Environment:**
 - OS: macOS High Sierra
 - Codegen: 0.14.5
 - Node: 8.11.1

**Comment by dotansimha** [comment]
Hi @DimaMukhin ! 
Until now we ignored default values defined in the GraphQL schema, because we couldn't reflect them in TypeScript interfaces. But it makes sense to declare this value as non-null if it has a default value.
We'll look into it soon. Thanks!

**Comment by kamilkisiela** [comment]
#1034

**Comment by dotansimha** [comment]
Fixed in 0.15.1
